style.css

```diff
@@ -117,6 +122,77 @@ body {
   margin-top: 0;
 }
 
+.menuDropDown {
+  position: fixed;
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  cursor: pointer;
+  opacity: 1;
+}
+
+.menu-cross {
+  opacity: 0;
+}
+
+.side-bar {
+  position: fixed;
+  top: 0;
+  left: 100%;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(25, 23, 101, 1);
+  background-repeat: no-repeat;
+  background-size: cover;
+  backdrop-filter: blur(5px);
+  background-image: url(bgimages/shapes\ mobile\ menu.png);
+  transition: left 0.5s ease;
+}
+
+.side-bar.active {
+  left: 0;
+}
+
+.menuDropDown.active {
+  opacity: 0;
+}
+
+.menu-cross.active {
+  opacity: 1 !important;
+  z-index: 1000;
+  position: fixed;
+  right: 26px;
+}
+
+.side-bar > .nav-li {
+  margin-top: 40px;
+  margin-left: 40px;
+}
+
+.entity {
+  padding: 24px 0;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 32px;
+  content: "/003E";
+  color: #191765;
+}
+
+.icons a {
+  color: #fff;
+}
+
+.nav-li > li > a {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 92%;
+}
+
+.fa-chevron-right {
+  color: #acb7c3 !important;
+}
```

Comment:
- [ ] Great job so far your hamburger menu button works as expected :clap: , however you've used another color to your icons, you mush follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins).:point_down:  kindly use the Figma design `color` for your icons as well as the `font-weight`

Your Design:

![iconss](https://user-images.githubusercontent.com/57562869/150996349-e6679ced-2dd3-4f51-b4f9-ebfaf73ad7f2.png)

Figma Design

![icon](https://user-images.githubusercontent.com/57562869/150996398-955fffee-846b-4f20-a2d1-339a806e1234.png)